docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -0,0 +1,419 @@
+---
+meta:
+  title: "Next.js Starter Kit"
+  parentTitle: "Guides"
+  description: "Get started with the Liveblocks Next.js Starter Kit"
+---
+
+## Features
+
+- Documents dashboard with pagination, drafts, groups, auto-revalidation
+- Collaborative whiteboard app with fully-featured share menu
+- Access can be granted to users, groups, and the public
+- Authentication compatible with GitHub, Google, Auth0, and more
+- Type-safe client & server functions, and everything documented
+- A great starting point for your collaborative application
+
+## Liveblocks
+
+This starter kit uses [Liveblocks](https://liveblocks.io]) to power its
+collaboration. Here's how to get it running.
+
+1. Create an account at [liveblocks.io](https://liveblocks.io])
```

Comment:
should we mention creating an application with liveblocks package installed

Reply:
or i guess will this get updated once its packaged?

Reply:
Ah, yes! 🙂 We should mention running the installer first. It'll probably look like this (it hasn't been published yet):
```
npx create-liveblocks-app --next
```